alliance/config/settings.py

```diff
@@ -112,10 +145,9 @@
 )
 
 PROJECT_APPS = (
-    'alliance.core',
-    'alliance.apps.backlog',
-    'alliance.apps.accounts',
-    'apps.shared'
+    'apps.shared',
+    'apps.backlog',
+    'apps.accounts',
 )
```

Comment:
Though I haven't tested this change to PROJECT_APPS yet (waiting for decision on the local_settings.py question), I'm almost positive this is going to fail in dev/prod on heroku. The reason is that paths get resolved there from one directory level *higher* than they do on our local django dev runtimes.

I think this is a pretty simple thing to accommodate - it just means re-introducing an environment specific settings file that overrides the master settings file, currently settings.py. Unlike local-settings.py, though, we want this file sourced.

We had this arrangement prior to this pull request in the form of dev.py which overrode and imported from base.py. Env var specified dev.py as the correct settings file for the dev/prod runtime. Can we go back to this approach?